spec/services/weather_man_spec.rb

```diff
@@ -43,22 +43,45 @@
   end
 
   describe ".outdoor_temp_for" do
+    it "only pulls from the NOAA central park weather station", :vcr do
+      time = Time.zone.parse("Feb 20, 2015 at 8am")
+      response = {
+        "history" => {
+          "observations" => [
+            {
+              "date" => {
+                "hour" => "8"
+              },
+              "tempi" => 30
+            }
+          ]
+        }
+      }
+      expect(WeatherMan.api).to receive(:history_for)
+        .with(time, 'knyc').and_return(response)
+      WeatherMan.outdoor_temp_for({
+        time: time,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.